site/plugins/site/src/Compressor/Cache.php

```diff
@@ -30,11 +32,4 @@ public function set(string $key, $value, int $minutes = 0): bool
         parent::set($key, $value, $minutes);
```

Comment:
```suggestion
        return parent::set($key, $value, $minutes);
```
Nothing was returning.